R/dvhydrograph-styles.R

```diff
@@ -13,6 +13,7 @@ getDvStyle <- function(data, info = NULL, ...){
                    est_stat2 = list(lines = list(x=x, y=y, type="s", col="red2", lty=3, pch=20, cex=0.5, legend.name=legend.name)),
                    est_stat3 = list(lines = list(x=x, y=y, type="s", col="red3", lty=6, pch=20, cex=0.5, legend.name=legend.name)),
                    est_comp = list(lines = list(x=x, y=y, type="s", col="red4", lty=6, pch=20, cex=0.5, legend.name=legend.name)),
+                   estEdges = list(arrows = list(x0=data[[1]]$x, x1=data[[1]]$x, y0=data[[1]]$y0, y1=data[[1]]$y1, lwd=1, lty=ifelse(data[[1]]$newSet == "est", 1, 2), col=ifelse(data[[1]]$newSet == "est", "blue", "red"), code=1, length = 0)),
```

Comment:
I like this implementation - in my head I was picturing adding a new point somewhere to make the lines connect, but this makes more sense.